it/repositories/SessionRepositoryISpec.scala

```diff
@@ -69,7 +69,7 @@ class SessionRepositoryISpec extends IntegrationSpecBase with MongoSupport with
     val mockConfiguration = mock[Configuration]
     val expireAfter = app.configuration.get[Int]("mongodb.timeToLiveInSeconds")
 
-    when(mockConfiguration.get[Boolean]("mongodb.replaceIndexes")).thenReturn(replaceIndexes)
+    when(mockConfiguration.get[String]("mongodb.replaceIndexes")).thenReturn(replaceIndexes.toString)
```

Comment:
Isn't this meant to be toBoolean now?

Reply:
That's in the tests; so it wants a String to be returned and it comes in as a Boolean (I know, it's a bit backwards)